ros/humble.v3.17/ros-humble-foonathan-memory-vendor/APKBUILD

```diff
@@ -0,0 +1,242 @@
+pkgname=ros-humble-foonathan-memory-vendor
+_pkgname=foonathan_memory_vendor
+pkgver=1.2.0
+pkgrel=0
+pkgdesc="$_pkgname package for ROS humble"
+url="https://index.ros.org/p/$_pkgname"
+arch="all"
+license="Apache License 2.0"
+
+depends="cmake ros-humble-ros-workspace"
+makedepends="chrpath cmake git py3-rosdep py3-rosinstall py3-rosinstall-generator py3-setuptools py3-vcstool ros-humble-ament-cmake-copyright ros-humble-ament-cmake-lint-cmake ros-humble-ament-cmake-test ros-humble-ament-cmake-xmllint"
```

Comment:
https://github.com/eProsima/foonathan_memory_vendor/blob/v1.2.0/package.xml
Originally `build_type` is specified as `cmake` but I changed it to `ament_cmake` because this package depends on `ros-humble-*` packages so the environment variables are needed.

Reply:
Addressed by https://github.com/alpine-ros/ros-abuild-docker/pull/158/commits/25992e37acc7e2d78da6d4210fc2ccec38a4e269